resources/install/protected/update_1.4.sql

```diff
@@ -58,7 +58,7 @@ CREATE TABLE IF NOT EXISTS `Downtime` (
   `creatorID` varchar(80) NOT NULL,
   `dateCreated` datetime NOT NULL,
   `startDate` datetime NOT NULL,
-  `endDate` datetime NOT NULL,
+  `endDate` datetime DEFAULT NULL,
```

Comment:
Because Resources 1.4 has just been released, which contained the initial Down Time features, this change needs to go in a separate upgrade script (presumably the 2.0 upgrade script, unless these changes are serious enough to release before then).